channeldb/channel.go

```diff
@@ -293,6 +295,12 @@ func (c *OpenChannel) UpdateCommitment(newCommitment *wire.MsgTx,
 		if err := putChanCapacity(chanBucket, c); err != nil {
 			return err
 		}
+		if err := putChanSatoshisSent(chanBucket, c); err != nil {
```

Comment:
Ah... thanks for the feedback! Upon scanning a few of the get/put/delete functions, I had thought they were mapped to one field per function (e.g. channelId, net fees, channel num), so I tried to do the same. Upon initial reading, I also thought the intention for putChanTotalFlow was to capture another field named totalFlow that would be the sum of satoshis sent and received, but it makes sense now that I've read it more closely. Maybe chanTotalFlow could be named something plural like put/fetch/deleteChanAmountsTransferred?

From a readability standpoint, I think it would be nice to at some point factor out the serialization code (ideally reducing to one parameterized put/fetch/delete function) so as to reduce noise and make the groupings more apparent.